next.config.js

```diff
@@ -3,16 +3,11 @@ const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
   compiler: { styledComponents: true },
+  images: {
+    unoptimized: true,
```

Comment:
this is just temporary right?

Reply:
There was a failure from the <Image> next component in Cloudflare for some reason, it suggested doing this to solve the issue. 

I'll address it later, not sure why the NextJs suggested component would cause a CI failure.